### DISCUSSION TITLE
Tab Key - Out of Range of Valid Values

### DISCUSSION THREAD

**Discussion by dyl20b** [SafeExamBrowser/seb-win-refactoring#540] [Q&A]
Hi,

I'm running into an issue when tabbing through the tabindex on SEB. After hitting the last tabindex, I get a "Fatal Error" message stating the specified argument was out of range of the valid values. However if I use [Shift]+[Tab] it will loop back around and not hit this out of range error.

Any ideas as to why this may be happening?
Thanks!

**Comment by dbuechel** [comment]
Could you please elaborate a bit further: You're encountering this issue with JavaScript code running within SEB?
